### DISCUSSION TITLE
Is this package still maintained?

### DISCUSSION THREAD

**Discussion by awdr74100** [nuxt-community/apollo-module#546] [General]
Some discussions indicate that the currently actively maintained package is [nuxt-graphql-client](https://github.com/Diizzayy/nuxt-graphql-client), Is this true? The author is also @Diizzayy.

**Comment by edgarsn** [comment]
Same thoughts. I would prefer this package over nuxt-graphql-client. For me this is kind a blocker in my projects before migrating to nuxt 3.

If current maintainers are tired of maintaining this package that's ok, it would be good to note that officially  to open doors for those who would like to help at this. 

This is a really good package for nuxt 2 currently and I hope it will not get deprecated.

**Comment by bmulholland** [comment]
https://github.com/Diizzayy/nuxt-graphql-client doesn't use Apollo. This is the one to use for Apollo. But yeah, it's quite broken and a big challenge for us upgrading.

**Comment by awacode21** [comment]
Anything new? it really looks unmantained. My current client is using this package but we are dealing with massive problems and i wonder if they will ever get fixed

**Reply by memic84** [reply]
Any more news on this?

**Reply by awacode21** [reply]
no.. we changed to URQL which works super smooth. no more apollo on our project

**Reply by memic84** [reply]
Thanks!
